ProjectEureka/project-eureka-flutter
# PE-71 | Created Route Names for Screens

**What I Did:** 

- Created a service file named `route_generator` that takes care of the navigation with screens without parameters. 
- Changed some filenames from page to screen to keep our file names uniform. 
- Removed unused imports. 

**How to Test:** 

- Uninstall the app from your emulator or physical phone. 
- Navigate through the app and make sure everything still works. Make sure to click on every button.
